implementations/rust/ockam/ockam_command/src/reset/static/after_long_help.txt

```diff
@@ -0,0 +1,13 @@
+```sh
+# Removes the local Ockam configuration including all Identities and Nodes.
+# This will prompt you for confirmation before proceeding.
+$ ockam reset
+
+# Removes orchestrator resources (i.e. Spaces and Projects)
+# in addition to deleting the local state.
+# This will prompt you for confirmation before proceeding.
```

Comment:
Same as above

Reply:
Thank you @adrianbenavides! I've noted your changes. I'll go ahead to resolve the conversations now.